### ISSUE TITLE
Add new ScanCode Package attributes to DataTables

### ISSUE THREAD

**Issue by jdaguil** [nexB/aboutcode-manager#81]
There are new package attributes that should be added to the table view

**Comment by mjherzog** [comment]
The most important new attributes for analysis are:
package_name
package_version
package_authors
package_homepage_url
package_download_urls
package_asserted_licenses

but it probably makes sense to add them all

**Comment by mjherzog** [comment]
We should have a Package Info choice for Column Visibility

**Comment by jdaguil** [comment]
@mjherzog I've added the subset of attributes you have listed, and I can merge that work into `develop`. I'm currently blocked by nexB/scancode-toolkit#617. I need additional information to add the other attributes correctly to our models and to the table view.

**Comment by jdaguil** [comment]
@mjherzog I will open a separate issue for adding the Package Info button

**Comment by jdaguil** [comment]
Merged into `develop`. Additional package specific attributes were taken care of in #68
